onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorUtility.cpp

```diff
@@ -132,6 +132,8 @@ namespace Dml
             std::string_view domain;
             int sinceVersion;
             std::vector<std::string_view> activationFilter;
+            bool enableOnMcdm;
```

Comment:
>enableOnMcdm [](http://example.com/codeflow?start=17&length=12)

Huh, that's weird the ORT should need such a low level detail, and that it's not just handled suitably in DirectML.dll. I'm not asking for changes from you, because I know this is already written by Jeff, but it's weird.

Reply:
Just talked with Jeff - it's a stopgap.